packages/gatsby/src/components/navigation.js

```diff
@@ -39,6 +39,7 @@ const MenuEntry = styled(Link)`
   align-items: center;
 
   text-decoration: none;
+  font-weight: 600;
```

Comment:
I don't find the bold menu very appealing

Reply:
Reverted